modules/consolidated-scanner.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="about-scanner-v4_{context}"]
+= About RHACS Scanner V4
+
+[role="_abstract"]
+{product-title-short} provides its own scanner, or you can configure an integration to use {product-title-short} with another vulnerability scanner.
+
+Beginning with release 4.4, Scanner V4, built on link:https://github.com/quay/claircore[ClairCore], is the default scanner for language and OS-specific image components. For earlier releases, you had to configure a separate integration to use Clair V4 as your scanner. A separate integration is no longer required for Clair scanning functionality. For release 4.4, {product-title-short} also uses the StackRox Scanner to provide some scanning functionality until that functionality is implemented in a future release.
```

Comment:
I was trying to get across the point that users now get Clair functionality built in, instead of having to configure a separate integration, but this is probably badly worded. I'll remove the integration part since that is covered in another area anyway.